### PR TITLE
Compile Sass via external pipeline using Sass CLI

### PR DESCRIPTION
**Pull request by tysongach**
Ruby Sass is deprecated and the sassc gem is unable to find Bourbon.
The turnover in Sass compilers in the Ruby community continues to
cause friction.

This switches over to pulling Sass (Dart Sass, specifically) from npm
and runs its CLI directly. This has the added benefit of using the most
modern Sass compiler.

https://sass-lang.com/documentation/cli/dart-sass

### CODE REVIEW

File: template/source/assets/stylesheets/main.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@use "normalize.css/normalize";
```
[the remark this answers is not in the document]
Comment (tysongach) [reply template/source/assets/stylesheets/main.scss:1]: [`@use` is a newer Sass feature](https://sass-lang.com/documentation/at-rules/use). Hound is using an old version of stylelint-scss, which is not aware of `@use`.

File: template/config.rb [hunk 2]
```diff
@@ -21,6 +21,15 @@
 page "/*.txt", layout: false
 page "/*.xml", layout: false
 
+ignore "*.scss"
+
+activate :external_pipeline, {
```
[the remark this answers is not in the document]
Comment (sneamonitakis) [reply template/config.rb]: I'm wondering if Hound is right about this one. In the Middleman documentation, it doesn't show the curly braces. https://middlemanapp.com/advanced/external-pipeline/

**Review by sneamonitakis — COMMENTED** [review]
Looks good to me!

**Review by sneamonitakis — APPROVED** [review]

File: template/config.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (tysongach) [reply template/config.rb]: I think you _can_, our RuboCop config just lints against them.

**Pull request merged**